modules/javafx.graphics/src/main/java/javafx/scene/Node.java

```diff
@@ -9037,13 +9104,13 @@ void completeTransitionTimers() {
 
         // Make a copy of the list, because completing the timers causes them to be removed
         // from the list, which would result in a ConcurrentModificationException.
-        for (TransitionTimer timer : List.copyOf(transitionTimers)) {
+        for (TransitionTimer timer : Map.copyOf(transitionTimers).values()) {
```

Comment:
If you only need a copy of the values, copying the whole map is a bit overkill.  How about `List.copyOf(transitionTimers.values())` ?  Lists are also much faster than maps for this purpose.

Reply:
Good idea.